ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/forms/emailTemplateFormUtils.ts

```diff
@@ -0,0 +1,31 @@
+import * as yup from 'yup';
+import { FormikProps } from 'formik';
+
+import {
+    ReportFormValues,
+    emailBodyValidation,
+    emailSubjectValidation,
+} from './useReportFormValues';
+
+// Helper functions
+
+export const defaultEmailSubjectTemplate =
+    'RHACS Workload CVE Report for <Config name>; Scope: <Collection name>';
+
+export const defaultEmailBody =
+    'Red Hat Advanced Cluster Security (RHACS) for Kubernetes has identified workload CVEs in the images matched by the following report configuration parameters. The attached Vulnerability report lists those workload CVEs and associated details to help with remediation. Please review the vulnerable software packages/ components from the impacted images and update them to a version containing the fix, if one is available.';
```

Comment:
@charmik-redhat @zhenpesky

Reply:
Talked to @charmik-redhat and he said, "The actual emails do use the correct branding based on the build. In case of email preview". So we can modify this as well.